### ISSUE TITLE
Including several editable pip installs generates a wrong requirements line

### ISSUE THREAD

**Issue by sdvillal** [ESSS/conda-devenv#118]
Example ```environment.devenv.yml```:

```yaml
dependencies:
  - pip:
    - "-e ./pado"
    - "-e ./pado_internal"
    - "-e ./pado_tggates"
```

Renders this ```environment.yml```:
```yaml
dependencies:
  - pip:
    - -e ./pado,./pado_internal,./pado_tggates
```

Which is incorrect and when run fails with an error like:
```ERROR: ./pado,./pado_internal,./pado_tggates is not a valid editable requirement.```

**Comment by nicoddemus** [comment]
Thanks @sdvillal for the report!

**Comment by sdvillal** [comment]
This actually seems to be fixed in 2.1.1 (I was running 2.1.0).
